### DISCUSSION TITLE
license badge

### DISCUSSION THREAD

**Discussion by eyllanesc** [cjolowicz/cookiecutter-hypermodern-python#1145] [Q&A] [answered]
Should the license badge point to `https://opensource.org/licenses/MIT` or `https://github.com/cjolowicz/cookiecutter-hypermodern-python-instance/blob/main/LICENSE`?

**Comment by cjolowicz** [comment, marked as answer]
Thanks for reporting this. The badge should link to the project license. Created #1147 for this.

**Reply by cjolowicz** [reply]
Admittedly, opensource.org provides additional context, which is nice.

But linking to the actual license seems the correct thing to do. After all, that text contains the names of the copyright holders.
